Doc/library/enum.rst

```diff
@@ -761,11 +761,11 @@ Data Types
 Supported ``__dunder__`` names
 """"""""""""""""""""""""""""""
 
-:attr:`__members__` is a read-only ordered mapping of ``member_name``:``member``
+:attr:`!__members__` is a read-only ordered mapping of ``member_name``:``member``
```

Comment:
Unless @ethanfurman has a reason to not explicitly document `__members__` but still list it here, we should just keep it as deliberately not resolving to avoid silencing an entirely valid warning, and also so that if and when it _is_ added, it seamlessly starts working here:

```suggestion
:attr:`~EnumType.__members__` is a read-only ordered mapping of ``member_name``:``member``
```